.goreleaser.yml

```diff
@@ -0,0 +1,98 @@
+project_name: migrate
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - 386
+    goarm:
+      - 7
+    main: ./cmd/migrate
+    ldflags:
+      - '-w -s -X main.Version={{ .Version }} -extldflags "static"'
+    flags:
+      - "-tags={{ .Env.DATABASE }} {{ .Env.SOURCE }}"
+      - "-trimpath"
+    ignore:
+      - goos: windows
+        goarch: arm
```

Comment:
Was windows arm causing issues?

Reply:
To be honest, that was in the original PR #570. Was thinking about to change it. Also going to add a format override for the windows archives, to have them as `zip`. ZIP seems to be more like a standard for windows.